README.md

```diff
@@ -1,3 +1,6 @@
+
+A collection of proofs of concept that demonstrate technology used throughout
+Instructure.
```

Comment:
lines 1 - 3 shouldn't be here. "# Test Advisory Board ..." should be the first line of this file.